tests/src/test/scala/com/evolutiongaming/skafka/FiberWithBlockingCancel.scala

```diff
@@ -0,0 +1,49 @@
+package com.evolutiongaming.skafka
+
+import cats.implicits._
+import cats.effect.syntax.all._
+import cats.effect.concurrent.Deferred
+import cats.effect.{Concurrent, Fiber, Resource, Timer}
+
+import scala.concurrent.duration.FiniteDuration
+
+object FiberWithBlockingCancel {
+  implicit class Ops[F[_], A](val self: F[A]) extends AnyVal {
```

Comment:
please add parent object's name as suffix in order to not have conflicts, hence `Ops` should be renames into something like `OpsFiberWithBlockingCancel`

Reply:
fixed in cc0db211